manual/googlechrome-extensions/googlechrome-github-expandinizr/googlechrome-github-expandinizr.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-github-expandinizr</id>
+    <version>1.0.0</version>
```

Comment:
true, it doesn't. But I thought part of chocolatey guidelines was to match upstream softwares version?